base_export_manager/models/res_users.py

```diff
@@ -11,6 +11,9 @@ class ResUsers(models.Model):
 
     @api.model
     def fetch_export_models(self):
+        """
+        Gets all models where the user has export access
+        """
```

Comment:
```suggestion
        """Gets all models where the user has export access"""
```

See https://www.python.org/dev/peps/pep-0257/#one-line-docstrings